src/components/ReportActionItem/TransactionPreview/types.ts

```diff
@@ -93,15 +93,19 @@ type TransactionPreviewContentProps = {
     /** Records any errors related to wallet terms. */
     walletTermsErrors: Errors | undefined;
 
-    /** Represents the IOU report entry from Onyx */
-    iouReport: OnyxEntry<Report>;
+    /** Represents the report linked to the transaction */
+    report: OnyxEntry<Report>;
 
     /** Flag to determine if a transaction involves a bill split among multiple parties. */
     isBillSplit: boolean;
 
     /** Holds the transaction data entry from Onyx */
     transaction: OnyxEntry<Transaction>;
 
+    /** The original amount value on the transaction. This is used to deduce who is the sender and who is the receiver of the money request
```

Comment:
I do not like the word `original` in this context as technically we have the modified amount, that is the changed/edited amount and `amount` is the original initially set amount so I think this could be slightly confusing to some  
```suggestion
    /** The amount of the transaction saved in the database. This is used to deduce who is the sender and who is the receiver of the money request
```

Reply:
Sounds good. We a few different versions of the amount so the nomenclature is tricky.
For split, if I split 10usd with you 5/5, then there exists both a transaction for `(+|-)500` and the transaction for full `1000`